platform/broadcom/libsaithrift-dev.dep

```diff
@@ -0,0 +1,13 @@
+#DPKG FRK
+SPATH       := $($(LIBSAITHRIFT_DEV)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) platform/broadcom/libsaithrift-dev.mk platform/broadcom/libsaithrift-dev.dep   
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+SMDEP_PATHS := $(SPATH)  $(SPATH)/bm/behavioral-model  $(SPATH)/test/ptf  $(SPATH)/test/saithrift/ctypesgen
+$(foreach path, $(SMDEP_PATHS), $(eval $(path) :=$(filter-out $(SMDEP_PATHS),$(addprefix $(path)/, $(shell cd $(path) &&  git ls-files)))))
```

Comment:
Build error as below, it may be caused by the space in file name, exclude the files as well?
sonic-buildimage/src/sonic-sairedis$ ls SAI/doc/OCP/
'SAI Demo 2016.pptx' 'SAI Demo 2017 topology.pdf'

12:10:19 make: *** No rule to make target 'src/sonic-sairedis/SAI/doc/OCP/SAI', needed by 'target/debs/stretch/libsaithrift-dev_0.9.4_amd64.deb.smdep'. Stop.
12:10:43 Makefile.work:209: recipe for target 'target/docker-ptf-brcm.gz' failed
12:10:43 make[1]: *** [target/docker-ptf-brcm.gz] Error 2

Reply:
Can you pull this PR as well
https://github.com/opencomputeproject/SAI/pull/1043

Reply:
it is hard to rename those files? can you just exclude the doc directory?

Reply:
Fixed